### DISCUSSION TITLE
@mckaygerhard ban

### DISCUSSION THREAD

**Discussion by objecttothis** [opensourcepos/opensourcepos#3669] [Team Posts]
Over the time that I have been contributing to this project, this user has pretty consistently been a nuisance and hasn't learned their lesson.  The last time this subject came up @daN4cat recommended that we use the GitHub features to ignore the user.  I did that, but it seems to have reverted because I'm back to seeing his vitriol.

I don't think I'm in a position to decide, but wanted to put forth the idea of banning him (https://help.github.com/en/github/building-a-strong-community/blocking-a-user-from-your-organization).  I know that he could potentially create another GitHub account and resume his antics, but I do think its better than nothing.  I believe the ban can be temporary, so we could start with that.  Thoughts?

**Comment by WebShells** [comment]
@objecttothis i agree, hes just messing every post/issue hes passing over...

**Comment by objecttothis** [comment]
@WebShells since @jekkos and @daN4cat call the shots, I'll leave it up to them.  I've made my thoughts known.

**Comment by jekkos** [comment]
He certainly seems to be less active.. but I agree that he is certainly a source of unnecessary noise on some of the threads. But I think that if we ban him, he will certainly become even more disgruntled and might try to seek for compensation for that action in another way. So not sure what is best.

**Comment by objecttothis** [comment]
> He certainly seems to be less active.. but I agree that he is certainly a source of unnecessary noise on some of the threads. But I think that if we ban him, he will certainly become even more disgruntled and might try to seek for compensation for that action in another way. So not sure what is best.

You think he might file a lawsuit, or did you mean compensation via trying to cause more strife?  I'm not a lawyer, but I think he definitely meets the criteria for being banned as it hasn't been on the basis of discrimination, but rather conduct.  I too, don't want to make the problem worse in trying to solve it, but I personally think it's worth the risk if the risk is that he could try to be more of a nuisance in posts through another account.  If that happens, we may be able to report him to Github and get all of his known accounts completely banned from Github not just this repository.

**Comment by WebShells** [comment]
> but I personally think it's worth the risk if the risk is that he could try to be more of a nuisance in posts through another account.

This is what i believe too...  He will try to evade the ban with another Username :/... 
Am not sure if he might count it as a serious warning

**Comment by SteveIreland** [comment]
I generally have a high tolerance for bad behavior due to the fact that I used to be a moderator for a Huntington's Disease Web site which involved trying to balance  providing support to those who have brain damage and bad behavior caused by brain damage.  I'll say that it was an interesting experience and a hell of a learning opportunity.

Over the last year I haven't been paying a lot of attention to what he's been posting. But for a brief period of time when he was actively being an asshole I was deleting his posts until he seemed to realize that his posts weren't going to be hanging around as a testimony to his "superior opinion".  That seemed to work but it only provided a brief respite.

But I'm also okay with banning him since there doesn't seem to be much hope for an attitude change on his part.  His posts often just add confusion and when the user starts listening to him, instead of us, I just stop paying attention to the issue.

**Comment by jekkos** [comment]
Ok let's try to ban him next time and then see what happens.

**Comment by objecttothis** [comment]
> Ok let's try to ban him next time and then see what happens.

OK, sounds good.  I think he's been quiet since I warned him, so hopefully it sticks.  If not, and he starts causing trouble we can at least ban him for a month before a permanent ban.
